demo/models.py

```diff
@@ -37,6 +43,9 @@ def has_newsletter_permission(self, user, action):
     @classmethod
     def get_newsletter_panels(cls):
         panels = [panel.clone() for panel in super().get_newsletter_panels()]
+        panels[-1:-1] = [
+            FieldPanel("newsletter_preview", heading="Preview"),
+        ]
```

Comment:
Why not use `panels.insert(-1, ...)`?

Reply:
Because it would seem that I'm slightly Python-illiterate 😅

I've used `panels.insert` as you suggest.